python/pylibvw.cc

```diff
@@ -1340,6 +1242,12 @@ BOOST_PYTHON_MODULE(pylibvw)
   py::class_<py_log_wrapper, py_log_wrapper_ptr>(
       "vw_log", "do not use, see pyvw.vw.init(enable_logging..)", py::init<py::object>());
 
+  py::class_<py_cpp_callback, py_cpp_callback_ptr>("pycpp_callback", py::no_init)
+      .def("call_base_learner", &py_cpp_callback::call_base_learner,
+          "Callback used for custom python reductions. See ReductionInterface. (you don't want to call this yourself!")
```

Comment:
Does `ReductionInterface` need to be updated?

Reply:
It's just a pointer to the interface they have to implement, and this is just a way of pushing the user to the right direction if they see this with python introspection or whatever docs get created by this side-effect of having python reductions (callback object)